backend/src/controller/donors.controllers.js

```diff
@@ -0,0 +1,156 @@
+const donorService = require("../services/donor.service.js");
+const Joi = require("joi");
+
+async function addDonor(req, res) {
+    // Joi validation
+    const schema = Joi.object().keys({
+        id: Joi.string().required(),
```

Comment:
why do we need to manage this id from frontend?

Reply:
removed id from frontend